test/test_operators.py

```diff
@@ -0,0 +1,283 @@
+    1d: 2pi * -sin(2pi x)
+    2d: 2pi * <cos(2pi x)cos(2pi y), -sin(2pi x)sin(2pi y)>
+    3d: 2pi * <cos(2pi x)sin(2pi y)cos(2pi z),
+               sin(2pi x)cos(2pi y)cos(2pi z),
+               -sin(2pi x)sin(2pi y)sin(2pi z)>
+    """
+    if x_vec is None:
+        return 0
+    dim = len(x_vec)
+    if grad:
+        ret_ary = make_obj_array([0*x_vec[0] + 1.0 for _ in range(dim)])
+        for i in range(dim):  # component & derivative for ith dir
+            for j in range(dim):  # form term for jth dir in ith component
+                if j == i:  # then this is a derivative term
+                    if j == (dim-1):  # deriv of cos term
+                        ret_ary[i] = ret_ary[i] * -actx.np.sin(2*np.pi*x_vec[j])
+                    else:  # deriv of sin term
+                        ret_ary[i] = ret_ary[i] * actx.np.cos(2*np.pi*x_vec[j])
+                    ret_ary[i] = 2*np.pi*ret_ary[i]
+                else:  # non-derivative term
+                    if j == (dim-1):  # cos term
+                        ret_ary[i] = ret_ary[i] * actx.np.cos(2*np.pi*x_vec[j])
+                    else:  # sin term
+                        ret_ary[i] = ret_ary[i] * actx.np.sin(2*np.pi*x_vec[j])
+    else:
+        # return _make_trig_term(actx, r=x_vec, term=dim-1)
+        ret_ary = actx.np.cos(2*np.pi*x_vec[dim-1])
+        for i in range(dim-1):
+            ret_ary = ret_ary * actx.np.sin(2*np.pi*x_vec[i])
+
+    return ret_ary
+
+
+def _cv_test_func(actx, x_vec, grad=False):
+    """Make a CV array container for testing.
+
+    There is no need for this CV to be physical, we are just testing whether the
+    operator machinery still gets the right answers when operating on a CV array
+    container.
+
+    Testing CV
+    ----------
+    mass = constant
+    energy = _trig_test_func
+    momentum = <_coord_test_func(order=2, fac=dim_index+1)>
+
+    Testing CV Gradient
+    -------------------
+    mass = <0>
+    energy = <_trig_test_func(grad=True)>
+    momentum = <_coord_test_func(grad=True)>
+    """
+    zeros = 0*x_vec[0]
+    dim = len(x_vec)
+    momentum = make_obj_array([zeros+1.0 for _ in range(dim)])
+    if grad:
+        dm = make_obj_array([zeros+0.0 for _ in range(dim)])
+        de = _trig_test_func(actx, x_vec, grad=True)
+        dp = make_obj_array([np.empty(0) for _ in range(dim)])
+        dy = (
+            momentum * make_obj_array([np.empty(0) for _ in range(0)]).reshape(-1, 1)
+        )
+        for i in range(dim):
+            dp[i] = _coord_test_func(actx, x_vec, order=2, fac=(i+1), grad=True)
+        return make_conserved(dim=dim, mass=dm, energy=de, momentum=np.stack(dp),
+                              species_mass=dy)
+    else:
+        mass = zeros + 1.0
+        energy = _trig_test_func(actx, x_vec)
+        for i in range(dim):
+            momentum[i] = _coord_test_func(actx, x_vec, order=2, fac=(i+1))
+        return make_conserved(dim=dim, mass=mass, energy=energy, momentum=momentum)
```

Comment:
The symbolic stuff is confusing to me.  I'm very slowly trying to grok it.

Reply:
Any chance you'd let this slide into a dedicated effort to replace this (and other) testing fixtures with symbolic constructs?  I really can't make heads or tails of it when I read it.

Reply:
This is the reason I think we should consider moving _away_ from the symbolic stuff in mirgecom (not add more of it).

Reply:
I dunno, I found the symbolic capabilities super helpful for avoiding messy algebra while writing the diffusion tests. IMO not hard to use for this once you understand the gist. Happy to give a quick tutorial @MTCam if you want.

Reply:
I re-implemented `_coord_test_func` symbolically as an example: #490. Want to take a stab at `_trig_test_func`? After that we can make some further simplifications (like computing gradients and evaluating expressions inside the test instead of inside each function).

Reply:
How's this? (#493)